packages/functional-tests/tests/subscription-tests/support.spec.ts

```diff
@@ -0,0 +1,88 @@
+import { test, expect } from '../../lib/fixtures/standard';
+
+test.describe('support form without valid session', () => {
+  test.beforeEach(() => {
+    test.slow();
+  });
+
+  test('go to support form, redirects to index', async ({
+    page,
+    target,
+    pages: { login },
+  }) => {
+    await login.clearCache();
+    await page.goto(`${target.contentServerUrl}/support`, {
+      waitUntil: 'networkidle',
+    });
+    expect(await login.isEmailHeader()).toBe(true);
+  });
+});
+
+test.describe('support form without active subscriptions', () => {
+  test.beforeEach(() => {
+    test.slow();
+  });
+
+  test('go to support form, redirects to subscription management, then back to settings', async ({
+    page,
+    target,
+    pages: { login },
+  }) => {
+    await page.goto(`${target.contentServerUrl}/support`, {
+      waitUntil: 'networkidle',
+    });
+    await page.waitForNavigation();
+    expect(await login.loginHeader()).toBe(true);
+  });
+});
+
+test.describe('support form with active subscriptions', () => {
+  test.beforeEach(() => {
+    test.slow();
+  });
+
+  test('go to support form, submits the form', async ({
+    pages: { login, relier, subscribe, settings, subscriptionManagement },
+  }) => {
+    await relier.goto();
+    await relier.clickSubscribe();
+    await subscribe.setFullName();
+    await subscribe.setCreditCardInfo();
+    await subscribe.clickPayNow();
+    await subscribe.submit();
+
+    //Login to FxA account
+    await login.goto();
+    await login.clickSignIn();
+    const subscriptionPage = await settings.clickPaidSubscriptions();
+    subscriptionManagement.page = subscriptionPage;
```

Comment:
Why is `page` set here? It doesn't appeared used below.

Reply:
This is because clicking on 'Paid Subscription' opens up a new page and to be able to work with that page and it's elements, the page needs to be set.